tests/app/main/test_permissions.py

```diff
@@ -38,7 +38,7 @@
         ("main.view_template", {"template_id": sample_uuid()}),
         ("main.view_template_versions", {"template_id": sample_uuid()}),
         ("main.view_template_version", {"template_id": sample_uuid(), "version": 1}),
-        ("no_cookie.view_letter_template_preview", {"template_id": sample_uuid(), "filetype": "pdf"}),
+        ("main.view_letter_template_preview", {"template_id": sample_uuid(), "filetype": "pdf"}),
```

Comment:
I think letter templates have been removed, you may wish to ensure that you're up to date with master/main